content/docs/getting-started/installation.md

```diff
@@ -57,17 +57,18 @@ in a terminal and see if the executable is found.
   cd ngspice
   sudo ./compile_linux.sh
   ```
-  Note that Ngspice has some dependencies that must be installed on the system.
-  Consult the [Ngspice website](https://ngspice.sourceforge.io/ for details regarding Ngspice installation.
+  Note that Ngspice has dependencies that must be installed on the system.
+  Consult the [Ngspice website](https://ngspice.sourceforge.io/) for details regarding Ngspice installation.
 
   For Windows a precompiled version is [available on the Ngspice website](https://ngspice.sourceforge.io/download.html).
+
   </i>
   </div>
 </div>
 
 ## Compilation
 
-Users can also compile OpenVAF themselves. 
+Users can also compile OpenVAF yourself. 
```

Comment:
themselves war hier richtig, weil wir in der 3. person sprechen (`Users` nicht `You`)

Reply:
ok, stimmt